dashboard/UI-Test/test.yaml

```diff
@@ -0,0 +1,65 @@
+sequence:
+- testcase 1
+- testcase 2
+test:
+- testcase 1:
+  - scenario: invalid pbench-dashboard login
+  - step 1:
+      action: goto
+      target: http://10.1.170.201
+      targets:
+      - http://10.1.170.201
```

Comment:
We don't want to hardcode IP addresses here; is there a way of importing these by variable from outside? This, in particular, is my personal test server and ideally nobody else would be depending on it at all ... in general, we'll be running our functional tests against a transient pod set up and torn down by the Jenkins pipeline and we need to run the dashboard tests against that server.